roles/matrix-ldap-registration-proxy/tasks/self_check_matrix_ldap_registration_proxy.yml

```diff
@@ -0,0 +1,22 @@
+---
+
+- ansible.builtin.set_fact:
+    matrix_ldap_registration_proxy_url_endpoint_public: "https://{{ matrix_server_fqn_matrix }}/_matrix/client/r0/register"
```

Comment:
This is exercising the homeserver more than the LDAP proxy though, right?

This API endpoint will always return stuff if the homeserver is up, regardless of whether it's wired to use LDAP proxy correctly or not.

Maybe removing this whole self-check thing is a good idea.